pkgs/applications/video/mpv/scripts/memo.nix

```
@@ -0,0 +1,34 @@
+{ fetchFromGitHub
+, buildLua
+, lib
+, unstableGitUpdater
+}:
+
+buildLua {
+  pname = "memo";
+  version = "unstable-2023-10-31";
+
+  src = fetchFromGitHub {
+    owner = "po5";
+    repo = "memo";
+    rev = "6f2b036ef860e130ea584657389e0a7ceb4d954f";
+    sha256 = "03rk1f97jblnajgwr36qz7hq1qlxrgkywc0gnq0wwg1vxiga9j4v";
+  };
+  passthru.updateScript = unstableGitUpdater {};
+
+  passthru.scriptName = "memo.lua";
+
+  installPhase = ''
+    mkdir -p $out/share/mpv/scripts
+    cp -r memo.lua $out/share/mpv/scripts
+  '';
+
+  meta = with lib; {
+    description = "A recent files menu for mpv.";
```

```suggestion
    description = "A recent files menu for mpv";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes